### ISSUE TITLE
[python_requires] won't load from 2nd remote repo

### ISSUE THREAD

**Issue by michaelmaguire** [conan-io/conan#4167]
Conan version 1.10.1  Windows

We had a ```conan_shared_python_requires/0.0.2@PORT/stable``` package in our internal ```internal_artifactory``` free-for-all repo.

Users were able to use it as expected.

I then uploaded it to our separate ```internal_artifactory_stable``` repo that is locked down, and removed it from our ```internal_artifactory``` free-for-all repo.

The user was then unable to conan create.

```
14:58:21 $ conan create . PORT/stable
conan_shared_python_requires/0.0.2@PORT/stable: Retrieving from predefined remote 'internal_artifactory'
conan_shared_python_requires/0.0.2@PORT/stable: Trying with 'internal_artifactory'...
ERROR: Unable to load conanfile in E:\DevelopBp\yji58\prepare_pal\pal\conanfile.py
```

This is despite the user being able to search and find the package in the ```internal_artifactory_stable``` repo:

```
conan search -r=internal_artifactory_stable
Existing package recipes:

[...]
conan_shared_python_requires/0.0.2@PORT/stable
```

To help us debug your issue please explain:

- [x] I've read the [CONTRIBUTING guide](https://raw.githubusercontent.com/conan-io/conan/develop/.github/CONTRIBUTING.md).
- [x] I've specified the Conan version, operating system version and any tool that can be relevant.
- [x] I've explained the steps to reproduce the error or the motivation/use case of the question/suggestion.

**Comment by michaelmaguire** [comment]
The user directly removed folders in the local cache.

Sorry we didn't realize that breaks things ;-)

**Comment by memsharded** [comment]
Yes, not always, but it can leave the cache in slightly inconsistent state. We have some initiative trying to avoid this, but at the moment, it is better to use ``conan remove`` commands.

**Comment by michaelmaguire** [comment]
Is there a way to fix

**Comment by memsharded** [comment]
Solved in 2.0, closing.
